item/paintings.lua

```diff
@@ -135,6 +135,22 @@ function M.UseItem(User, SourceItem)
             "The picture begins to glow slightly and one can see that the potion inside the cauldron turns a dark green and an evil�skull�rises up out of the cauldon.")
         return
     end
+	local pictureno=SourceItem:getData("dickerquest");
+	if tonumber(pictureno) and User:getQuestProgress(674) == 100 then
```

Comment:
`toNumber` is very likely wrong in this case. Because that function converts the value to number. So this statement is true for any number not equal 0. If that is what you wanted please write it as a clear compare function for easier readability.